2-extra/1-choose-your-own-adventure.js

```diff
@@ -44,6 +44,13 @@ let game = {
     // object for the correct room.
     //
     // Hint: the only valid rooms are "hall", "classroom" and "library".
+    if (roomName == "hall"){
```

Comment:
It took me a while to understand this but the key is that `game.currentRoom` has a value from `rooms`. With that information, you should be able to get this by using array and object methods. 

I would review the object built-in methods from the lesson: https://syllabus.codeyourfuture.io/js-core-2/week-2/lesson#object-built-in-methods